WebContent/js/containers/Filters.js

```diff
@@ -165,10 +166,13 @@ export class Filters extends React.Component {
                     id="filter-view-header"
                     expandIcon={<ExpandMoreIcon />}
                     onClick={() => { this.toggleFilters(); }}
+                    ref={this.filterPanelRef}
```

Comment:
Is this ref used anywhere?

Reply:
was experimenting, removed it, thanks for catching this.